src/App.tsx

```diff
@@ -24,14 +24,63 @@ const footerStyles = css`
   justify-content: flex-end;
 `;
 
+const DEFAULT_FILTERS: FilterOptions = {
+  orderBy: "newest",
+  pageSize: 25,
+  threads: "collapsed",
+  page: 1
+};
+
+const rememberFilters = (filtersToRemember: FilterOptions) => {
+  try {
+    localStorage.setItem(
```

Comment:
Sorry, know this is merged but @oliverlloyd @liywjl I had a thought last night that we should try and think about how we create an abstraction layer over the *browser* specific things - like localstorage, so that they can hook into other technologies - apps for example might want to store these in the native layer so we should define a contract where we can plug in localstorage but others can plugin whatever mechanism works for them. No need to change this just yet but something to think about when building!

Reply:
Yep, I agree (I made a comment about this ^). Probably worth reviewing FE to see what learnings are available and possibly having a look at libs like localForage to see how they handle edge cases.

Reply:
oh yeh. cool!